lib/mongoose_push.ex

```diff
@@ -140,9 +140,10 @@ defmodule MongoosePush do
 
   defp emit_telemetry_event(time, :ok, service, mode) do
     :telemetry.execute(
-      [:mongoose_push, service, :push, :success],
+      [:mongoose_push, :push],
```

Comment:
Since we're changing the event name, can we change it to `[:mongoose_push, :notification, :send]`. I feel like this is more telling ;)